packages/building-rollup/src/createSpaConfig.js

```diff
@@ -72,30 +105,7 @@ function createSpaConfig(options) {
       pluginWithOptions(polyfillsLoader, userOptions.polyfillsLoader, polyfillsLoaderConfig),
 
       // generate service worker
-      userOptions.workbox &&
-        pluginWithOptions(
-          generateSW,
-          userOptions.workbox,
-          {
-            globIgnores: ['polyfills/*.js', 'legacy-*.js', 'nomodule-*.js'],
-            navigateFallback: '/index.html',
-            // where to output the generated sw
-            swDest: path.join(process.cwd(), outputDir, 'sw.js'),
-            // directory to match patterns against to be precached
-            globDirectory: path.join(process.cwd(), outputDir),
-            // cache any html js and css by default
-            globPatterns: ['**/*.{html,js,css,webmanifest}'],
-            skipWaiting: true,
-            clientsClaim: true,
-            runtimeCaching: [
-              {
-                urlPattern: 'polyfills/*.js',
-                handler: 'CacheFirst',
-              },
-            ],
-          },
-          () => {},
-        ),
+      userOptions.workbox && workboxPlugin,
```

Comment:
This should not be needed, right? Because you already push the workboxPlugin to the plugins array on line 95?

Reply:
Ops! I forget to remove, yes!